### ISSUE TITLE
Unable to start v8_latest(v8_0_17) on macOs 10.14.6

### ISSUE THREAD

**Issue by BrainLogic** [wix/wix-embedded-mysql#182]
```
17:19:32.659 [ScalaTest-run] INFO com.wix.mysql.EmbeddedMysql - Preparing EmbeddedMysql version 'Version 8.0.17'...
Download Version 8.0.17:OS_X:B64 START
Download Version 8.0.17:OS_X:B64 DownloadSize: 155365544
...
Download Version 8.0.17:OS_X:B64 DONE
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz START
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/bin/mysql
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/bin/mysqladmin
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/bin/my_print_defaults
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/bin/mysqld
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/share/english/errmsg.sys
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/lib/libcrypto.1.0.0.dylib
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz extract mysql-8.0.17-macos10.14-x86_64/lib/libssl.1.0.0.dylib
Extract /Users/%username%/.embedmysql/MySQL-8.0/mysql-8.0.17-macos10.14-x86_64.tar.gz DONE
17:19:49.621 [ScalaTest-run] ERROR de.flapdoodle.embed.process.runtime.Starter - no message
java.nio.file.NoSuchFileException: /Users/%username%/.embedmysql/extracted/OS_X-B64--/MySQL-8.0/mysql-8.0.17-macos10.14/bin/resolveip
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixCopyFile.copy(UnixCopyFile.java:526)
	at sun.nio.fs.UnixFileSystemProvider.copy(UnixFileSystemProvider.java:253)
	at java.nio.file.Files.copy(Files.java:1274)
	at de.flapdoodle.embed.process.extract.ExtractedFileSets.copy(ExtractedFileSets.java:62)
	at de.flapdoodle.embed.process.store.ExtractedArtifactStore.extractFileSet(ExtractedArtifactStore.java:110)
	at com.wix.mysql.store.SafeExtractedArtifactStore.extractFileSet(SafeExtractedArtifactStore.java:30)
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:57)
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:49)
	at com.wix.mysql.EmbeddedMysql.<init>(EmbeddedMysql.java:41)
	at com.wix.mysql.EmbeddedMysql$Builder.start(EmbeddedMysql.java:169)
	at com.verizon.sensity.lighting.etl.db.SchemaMigrationSpec.beforeAll(SchemaMigrationSpec.scala:27)
	at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:212)
	at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
	at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
	at com.verizon.sensity.lighting.etl.db.SchemaMigrationSpec.run(SchemaMigrationSpec.scala:13)
	at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)
	at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1349)
	at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1343)
	at scala.collection.immutable.List.foreach(List.scala:392)
	at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1343)
	at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:1033)
	at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:1011)
	at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1509)
	at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:1011)
	at org.scalatest.tools.Runner$.run(Runner.scala:850)
	at org.scalatest.tools.Runner.run(Runner.scala)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.runScalaTest2(ScalaTestRunner.java:133)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.main(ScalaTestRunner.java:27)
Exception encountered when invoking run on a nested suite - java.nio.file.NoSuchFileException: /Users/%username%/.embedmysql/extracted/OS_X-B64--/MySQL-8.0/mysql-8.0.17-macos10.14/bin/resolveip
de.flapdoodle.embed.process.exceptions.DistributionException: java.nio.file.NoSuchFileException: /Users/%username%/.embedmysql/extracted/OS_X-B64--/MySQL-8.0/mysql-8.0.17-macos10.14/bin/resolveip
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:69)
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:49)
	at com.wix.mysql.EmbeddedMysql.<init>(EmbeddedMysql.java:41)
	at com.wix.mysql.EmbeddedMysql$Builder.start(EmbeddedMysql.java:169)
	at com.verizon.sensity.lighting.etl.db.SchemaMigrationSpec.beforeAll(SchemaMigrationSpec.scala:27)
	at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:212)
	at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
	at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
	at com.verizon.sensity.lighting.etl.db.SchemaMigrationSpec.run(SchemaMigrationSpec.scala:13)
	at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)
	at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1349)
	at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1343)
	at scala.collection.immutable.List.foreach(List.scala:392)
	at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1343)
	at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:1033)
	at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:1011)
	at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1509)
	at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:1011)
	at org.scalatest.tools.Runner$.run(Runner.scala:850)
	at org.scalatest.tools.Runner.run(Runner.scala)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.runScalaTest2(ScalaTestRunner.java:133)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.main(ScalaTestRunner.java:27)
Caused by: java.nio.file.NoSuchFileException: /Users/%username%/.embedmysql/extracted/OS_X-B64--/MySQL-8.0/mysql-8.0.17-macos10.14/bin/resolveip
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixCopyFile.copy(UnixCopyFile.java:526)
	at sun.nio.fs.UnixFileSystemProvider.copy(UnixFileSystemProvider.java:253)
	at java.nio.file.Files.copy(Files.java:1274)
	at de.flapdoodle.embed.process.extract.ExtractedFileSets.copy(ExtractedFileSets.java:62)
	at de.flapdoodle.embed.process.store.ExtractedArtifactStore.extractFileSet(ExtractedArtifactStore.java:110)
	at com.wix.mysql.store.SafeExtractedArtifactStore.extractFileSet(SafeExtractedArtifactStore.java:30)
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:57)
	... 21 more

```

**Comment by BrainLogic** [comment]
There is no `resolveip` in `bin` dir of `mysql-8.0.17-macos10.14-x86_64.tar.gz`

**Comment by soaserele** [comment]
Hi @BrainLogic. Yes, you are right, there is no `bin/resolveip` file in the MySQL 8.0.17 archive. We had the same issue and had to patch the `wix-embedded-mysql` code. You can find the patch attached in the comments at the #181 issue.

**Comment by viliusl** [comment]
@BrainLogic - fix released in `4.6.1`. Closing issue for now, please let me know if you still have some issues with 8.0.17
